arch/arm/core/cortex_m/swap_helper.S

```diff
@@ -298,8 +429,7 @@ _stack_frame_endif:
 #endif
 
     /* exception return is done in z_arm_int_exit() */
-    ldr r0, =z_arm_int_exit
-    bx r0
+    b z_arm_int_exit
```

Comment:
This hunk needs to be restored.

https://github.com/zephyrproject-rtos/zephyr/pull/80765

Reply:
@nashif will you push that extra commit into this branch?